sr/tools/cli/inv_mv.py

```
@@ -61,6 +71,18 @@ def add_subparser(subparsers):
         help="If the asset codes are part of an assembly then "
         "move the whole assembly.",
     )
+    parser.add_argument(
+        "-q",
+        "--ignore-assy",
+        action="store_true",
+        help="Don't print warnings about items being part of an assembly.",
+    )
+    parser.add_argument(
+        "--ignore-invalid",
```

Question: What's the use case behind `--ignore-invalid`? `--ignore-assy` I get, since it's verbose, but Ignoring errors sounds like it's hiding a different issue.

---

Without `--ignore-invalid` the command just fails on the first invalid code and nothing moves. With it you get an error line for each of the invalid codes and everything else moves.

Without `--ignore-assy` you get loads of warnings about various items being parts of assemblies, which we no longer particularly make use of. This makes it more likely to miss that a move failed, for instance:

> Warning: Part 0HBR is in an assembly. To move the assembly, use the -a switch.
Warning: Part 0NU2L is in an assembly. To move the assembly, use the -a switch.
Warning: Part 0YF3 is in an assembly. To move the assembly, use the -a switch.
Warning: Part 0PT2K is in an assembly. To move the assembly, use the -a switch.
Warning: Part G2UAW is in an assembly. To move the assembly, use the -a switch.
Warning: Part 0TDC is in an assembly. To move the assembly, use the -a switch.
Warning: Part Y1H2A is in an assembly. To move the assembly, use the -a switch.
Warning: Part M1K94 is in an assembly. To move the assembly, use the -a switch.
Warning: Part N1H4F is in an assembly. To move the assembly, use the -a switch.
Warning: Part M1J95 is in an assembly. To move the assembly, use the -a switch.
Warning: Part G2RAY is in an assembly. To move the assembly, use the -a switch.
Warning: Part 0REB is in an assembly. To move the assembly, use the -a switch.
Error: DQK1T is an invalid asset code.

This output is a command that failed so nothing was moved, but if you miss the last line, then this is just expected output for a successful command.